sumo_deployments/helm/sumo-opentelemetry-demo/values.yaml

```diff
@@ -130,6 +133,21 @@ opentelemetry-demo:
         affinity: {}
         tolerations: []
 
+    accountingService:
+      enabled: true
+      useDefault:
+        env: true
+      env:
+        - name: OTEL_EXPORTER_OTLP_ENDPOINT
+          value: 'http://{{ include "sumo-opentelemetry-demo-otelagent-svc-name" . }}:4317'
```

Comment:
Does this even work? Do you manually handle it somewhere?

Reply:
Yes, it works fine and it's implemented [here](https://github.com/SumoLogic/opentelemetry-demo/blob/add-sumo-rum-frontend/sumo_deployments/helm/sumo-opentelemetry-demo/templates/_helpers.tpl#L65)